xbmc/cores/AudioEngine/Utils/AEStreamInfo.cpp

```diff
@@ -446,6 +446,25 @@ bool CAEStreamParser::TrySyncAC3(uint8_t *data, unsigned int size, bool resyncin
     m_fsize = framesize << 1;
     m_info.m_repeat = MAX_EAC3_BLOCKS / blocks;
 
+    /* EAC3 can have a dependent stream too */
```

Comment:
Whilst I really, really hate to pick nits here - you've just carefully converted every other comment in this file to //

Reply:
I'm not following. This commit follows the formatting of the original code (as it's basically a copy and paste).
The next two commits apply the code formatting guidelines to the whole file.

Reply:
Oh - mea culpa - I misread the commit order